cpp/src/gandiva/precompiled/CMakeLists.txt

```diff
@@ -27,29 +41,24 @@ set(PRECOMPILED_SRCS
     time.cc
     timestamp_arithmetic.cc
     ../../arrow/util/basic_decimal.cc)
-set(GANDIVA_PRECOMPILED_BC_FILES)
-foreach(SOURCE ${PRECOMPILED_SRCS})
-  gandiva_add_bitcode(${SOURCE})
-  get_filename_component(SOURCE_BASE ${SOURCE} NAME_WE)
-  list(APPEND GANDIVA_PRECOMPILED_BC_FILES ${CMAKE_CURRENT_BINARY_DIR}/${SOURCE_BASE}.bc)
-endforeach()
+generate_and_link_bitcode("${PRECOMPILED_SRCS}" ${GANDIVA_PRECOMPILED_BC_PATH})
 
-# link all of the bitcode files into a single bitcode file.
-add_custom_command(OUTPUT ${GANDIVA_PRECOMPILED_BC_PATH}
-                   COMMAND ${LLVM_LINK_EXECUTABLE} -o ${GANDIVA_PRECOMPILED_BC_PATH}
-                           ${GANDIVA_PRECOMPILED_BC_FILES}
-                   DEPENDS ${GANDIVA_PRECOMPILED_BC_FILES})
+set(MANDATORY_SRCS bitmap.cc)
```

Comment:
So far, all the mandatory IR functions, including "bitMapGetBit"/"bitMapSetBit"/"bitMapValidityGetBit"/"bitMapClearBitIfFalse", are defined in `bitmap.cc` file